README.md

```diff
@@ -1,5 +1,5 @@
 [![Integration tests](https://github.com/argoproj/argo-cd/workflows/Integration%20tests/badge.svg?branch=master)](https://github.com/argoproj/argo-cd/actions?query=workflow%3A%22Integration+tests%22)
-[![slack](https://img.shields.io/badge/slack-argoproj-brightgreen.svg?logo=slack)](https://argoproj.github.io/community/join-slack)
+[![slack](https://img.shields.io/badge/slack-argo_cd-brightgreen.svg?logo=slack)](https://cloud-native.slack.com/archives/C01TSERG0KZ)
```

Comment:
Is that actually the correct URL? If I click on that, I get a sign on page, but no possibility to create an account :thinking:

Reply:
I think we've posted `https://argoproj.github.io/community/join-slack` in other places. 

Can we update https://github.com/argoproj/argo-site/blob/master/static/community/join-slack.html instead and keep using `https://argoproj.github.io/community/join-slack` ?

Reply:
@jannfis Good catch...

@alexmt Perhaps we can update that join-slack html to be a plain HTML instead of a redirect? The content would include instruction to join https://slack.cncf.io/ workspace and then provide links to different channels on that page. WDYT?

Reply:
I discussed this with @alexmt who's going to reuse the existing link with instructions to join the CNCF workspace and channels. Closing this for now.